src/stk/molecular/topology_graphs/host_guest/complex.py

```diff
@@ -105,19 +153,24 @@ def __init__(
         host : :class:`.BuildingBlock`
             The host molecule.
 
-        guest : :class:`.BuildingBlock`
-            The guest molecule.
+        guests : :class:`tuple` of :class:`.Guest`, optional
+            The guest molecules.
+
+        guest : :class:`.BuildingBlock`, optional
+            The guest molecule. This is the old API.
```

Comment:
I would say something like

```
The guest molecule. This API is deprecated and will be removed in any :mod:`.stk` version 
released on, or after, 29/12/2021. Use the `guests` parameter instead. 
```

Add a disclaimer of this form to the other deprecated parameters.